### PR TITLE
BEEFY update CODEOWNERS

### PR DESCRIPTION
**Pull request by adoerr**
Remove myself as `BEEFY` code owner.  New code owner needs to be decided upon.

### CODE REVIEW

File: docs/CODEOWNERS [hunk 1]
```diff
@@ -41,10 +41,10 @@
 /primitives/consensus/pow/ @sorpaas
 
 # BEEFY
```
Comment (bkchr) [inline docs/CODEOWNERS:43]:
```suggestion
```

File: docs/CODEOWNERS [hunk 2]
```diff
@@ -41,10 +41,10 @@
...
-/client/beefy/ @adoerr
-/frame/beefy/ @adoerr
-/frame/beefy-mmr/ @adoerr
-/primitives/beefy/ @adoerr
+#/client/beefy/
+#/frame/beefy/ 
+#/frame/beefy-mmr/ 
+#/primitives/beefy/
```
Comment (bkchr) [inline docs/CODEOWNERS:47]:
```suggestion
```

**Review by bkchr — APPROVED** [review]

File: docs/CODEOWNERS [hunk 3]
```diff
@@ -40,11 +40,6 @@
 /client/consensus/pow/ @sorpaas
 /primitives/consensus/pow/ @sorpaas
```
Comment (bkchr) [inline docs/CODEOWNERS:42]:
```suggestion
```

**Comment by tomusdrw** [comment]
CC @acatangiu perhaps you'd like to take over the ownership?

**Comment by acatangiu** [comment]
yes, will do :+1:

**Pull request merged**